### ISSUE TITLE
Error loading up Sia-IU

### ISSUE THREAD

**Issue by redspy27** [NebulousLabs/Sia#1895]
A critical error loading Sia has occured: Siad unexpectedly exited. Error log: Loading...
(0/6) Loading siad...
(1/6) Loading gateway...
(2/6) Loading consensus...
(3/6) Loading transaction pool...
(4/6) Loading wallet...
(5/6) Loading host...
Closing wallet...
Closing transaction pool...
Closing consensus set...
Closing gateway...
unable to read persisted json object from disk: unable to read header from persisted json object file: EOF

// Just started trying to mine today but I have encountered many issues all my own fault up to this point. I can not seen to figure out what is wrong and what needs to be done to fix it.

-Using the newest version 1.2.2
-tried re-downloading it already  

Using windows 10 x64

**Comment by redspy27** [comment]
A critical error loading Sia has occured: Siad unexpectedly exited. Error log: Loading...
(0/6) Loading siad...
(1/6) Loading gateway...
(2/6) Loading consensus...
(3/6) Loading transaction pool...
(4/6) Loading wallet...
(5/6) Loading host...
Closing wallet...
Closing transaction pool...
Closing consensus set...
Closing gateway...
unable to read persisted json object from disk: unable to read header from persisted json object file: EOF

// Just started trying to mine today but I have encountered many issues all my own fault up to this point. I can not seen to figure out what is wrong and what needs to be done to fix it.

-Using the newest version 1.2.2
-tried re-downloading it already

Using windows 10 x64
